File: 2592.py
S = []
for i in range(0,10):
    S += [input()]
Max = S[0]
length = 1
record = 1
for x in S[1:]:
    if x == Max:
        length += 1
        record += 1
    else :
        length = 1
    if record < length:
        record = length
        Max = x
sum = 0
for i in S:
    sum += int(i)
sum /= 10
print(int(sum))
print(Max)

File: C_C++/2592.cpp
#include <cstdio>

int inp[1002];

int main() {
  int sum = 0;
  for( int i = 0; i < 10; i++){
    int N; scanf("%d", &N); inp[N]++;
    sum += N;
  }
  printf("%d\n", sum / 10);
  int Max = 0, idx;
  for ( int i = 0; i < 1001; i += 10){
    if (inp[i] > Max){
      Max = inp[i];
      idx = i;
    }
  }
  printf("%d\n", idx);
}
